pkg/ipmi/ipmi.go

```diff
@@ -88,6 +94,34 @@ type systemInterfaceAddr struct {
 	lun      byte //nolint:unused
 }
 
+type StandardEvent struct {
+	Timestamp    uint32
+	GenID        uint16
+	EvMRev       uint8
+	SensorType   uint8
+	SensorNum    uint8
+	EventTypeDir uint8
+	EventData    [3]uint8
+}
+
+type OEMTsEvent struct {
```

Comment:
customarily, each exported type and function should have a comment explaining it. the comment should be a full sentence starting with just the name of the exported type, e.g.

```go
// OEMTsEvent is a timestamped OEM-custom event.
//
// It holds 6 bytes of OEM-defined arbitrary data.
```